my-app/functions-python/main.py

```
@@ -7,6 +7,10 @@
     calculate_optimal_cluster_route,
     geocode_addresses_endpoint
 )
+import os
+from mailjet_rest import Client
+from datetime import datetime, timedelta
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)